src/server/project.ts

```diff
@@ -334,6 +375,47 @@ namespace ts.server {
             }
         }
 
+        getReferencedFiles(fileName: string): string[] {

```

Comment:
switch from `string` to `Path` to avoid extra internal conversion steps. At the call site `Path` can be obtained from the `ScriptInfo.path` property